plugins/python-build/bin/python-build

```diff
@@ -156,7 +156,7 @@ num_cpu_cores() {
     num="$(sysctl -n hw.ncpu 2>/dev/null || true)"
     ;;
   * )
-    num="$({ getconf _NPROCESSORS_ONLN ||
+    num="$({ getconf _NPROCESSORS_ONLN || getconf NPROCESSORS_ONLN ||
```

Comment:
The `*` case looks like it is assuming the system is very similar to Linux.... To add support for Solaris/Illumos, I prefer to add a case condition for Solaris.

Reply:
Sure - I'll add a SunOS case statement instead.